api/desecapi/crypto.py

```diff
@@ -5,10 +5,11 @@
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives.kdf.kbkdf import CounterLocation, KBKDFHMAC, Mode
 from cryptography.hazmat.backends import default_backend
+from desecapi import metrics
```

Comment:
we usually keep all desecapi imports in a separate block after all other imports